content/docs/examples/relationaldb.md

```diff
@@ -21,7 +21,7 @@ Aurora Databases are designed to integrate with Serverless environments by execu
 {{<break 1>}}
 ## Creating a Relational DB
 {{<codesnippet `/source_code/relationaldb_examples/basic_db.py`>}}
-{{<tool_tip key="tip" summary="Database Info">}}
```

Comment:
Hi Jordan, sorry if I was not clear. --- line 24....change `key="tip"`, to `key='info`.